.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,33 @@
-## Testing
+## ℹ️ Description
 
 <!--
-What is needed to test the changes? e.g. specific cluster, service definitions
-How can one see the result of your work? e.g. configurations, URLs
+What is the change your PR does?
+Please add a short description here. (one sentence is enough)
 -->
 
-## Trade-offs
+- 🔖 JIRA: [DCOS-12345](https://jira.mesosphere.com/browse/DCOS-12345)
+  <!-- add things that reviewers need to know, e.g.
+- 🚫 This blocks #PRNumber
+- ⚠️ Component is not enabled yet (see testing)
+- ⚠️ Wasnt able to test it yet, cluster is still starting… 😅
```

Comment:
Then you shouldn't open a PR up for review :trollface:

Reply:
example and what is important in a particular PR decides the creator 🤷‍♂️

Reply:
But you chose the example, therefore it carries a message / weight as an example.

Reply:
I can remove it.